src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -113,10 +117,7 @@ async fn run() {
 
             let mut options_with_tls = options.clone();
             if requires_tls {
-                let tls_options = TlsOptions::builder()
-                    .allow_invalid_certificates(true)
-                    .build();
-                options_with_tls.tls = Some(Tls::Enabled(tls_options));
+                options_with_tls.tls = CLIENT_OPTIONS.tls.clone();
```

Comment:
What's the reasoning behind this change? From looking over DRIVERS-1324 it seems like we still need to be running these tests with TLS enabled.

Reply:
This ensures the test respects the TLS settings specified in the `MONGODB_URI` environment variable. The previously hard-coded TLS options here happened to work with our evergreen config, but it didn't work when I was testing locally, for example, but the new config works in both places. Also, on line 94 we verify that TLS is enabled in `CLIENT_OPTIONS` so copying it here also enables it.